lib/Dialect/LWE/IR/LWEDialect.cpp

```diff
@@ -34,6 +34,25 @@ namespace mlir {
 namespace heir {
 namespace lwe {
 
+class LWEOpAsmDialectInterface : public OpAsmDialectInterface {
+ public:
+  using OpAsmDialectInterface::OpAsmDialectInterface;
+
+  AliasResult getAlias(Type type, raw_ostream& os) const override {
+    auto res = llvm::TypeSwitch<Type, AliasResult>(type)
+                   .Case<RLWECiphertextType>([&](auto& type) {
+                     os << "rlwe_ct";
+                     return AliasResult::OverridableAlias;
+                   })
+                   .Case<RLWEPlaintextType>([&](auto& type) {
+                     os << "rlwe_pt";
+                     return AliasResult::OverridableAlias;
```

Comment:
Is there a reason these are overridable while the others are final?

Reply:
>  enum class AliasResult {
    /// The object (type or attribute) is not supported by the hook
    /// and an alias was not provided.
    NoAlias,
    /// An alias was provided, but it might be overriden by other hook.
    OverridableAlias,
    /// An alias was provided and it should be used
    /// (no other hooks will be checked).
    FinalAlias
  };
  
I thought that if two different RLWECiphertextType has the same alias with `FinalAlias`, MLIR would complain, as by `(no other hooks will be checked)` it wont assign number to them. To reply this comment I expreimented with using `FinalAlias`, found that MLIR still assign number to them. I think we can all use `FinalAlias` safely.